pytorch_lightning/trainer/optimizers.py

```diff
@@ -117,6 +117,13 @@ def configure_schedulers(self, schedulers: list, monitor: Optional[str] = None):
                     raise MisconfigurationException(
                         'The lr scheduler dict must have the key "scheduler" with its item being an lr scheduler'
                     )
+                allowed_interval_keys = ['step', 'epoch']
+                if 'interval' in scheduler and scheduler['interval'] not in allowed_interval_keys:
```

Comment:
```suggestion
                if 'interval' in scheduler and scheduler['interval'] not in ('step', 'epoch'):
```

Reply:
`allowed_interval_keys` is used in error message. need to revert the changes here I guess. @ddrevicky

Reply:
Oops, my bad 🙈